tests/integration/test_compat_mode.py

```diff
@@ -75,13 +75,13 @@ async def test_blocked_tasks_in_n0(flow, scheduler, run, complete):
         } == {
             ('1', 1, 'foo'),
             ('1', 0, 'recover'),
-            ('1', 0, 'bar'),  # NOTE: this should be n=2 not n=0
```

Comment:
This should not have been `n=2` it is one edge away from active task `recover`..